src/components/drawer/FamilyMatchesDrawer.tsx

```diff
@@ -27,6 +28,12 @@ export const FamilyMatchesDrawer = ({ family }: TProps) => {
     router.push({ pathname: `/documents/${document.document_slug}`, query: queryObj });
   };
 
+  const showPageNumbers = (documentContentType: TDocumentContentType) => {
+    if (!documentContentType) return null;
```

Comment:
I'm intrigued, why are we returning null?

Reply:
Changed to explicitly return false instead of falsey null.